### ISSUE TITLE
Problem with this code

### ISSUE THREAD

**Issue by MDP-DL-UNISA** [MDP-DL-UNISA/Custom-Pooling-Layer#1]
Hi guys.
As explained in the description, I tried to implement a custom pooling layer using Keras. In particular, I'm trying to implement Choquet integral as Pooling through a sliding window upon the input filters. 
So at this link:
(https://colab.research.google.com/drive/1agwh8lUPntGQazxpee27bxjJpGrcssJB)
you could find a Colab Notebook to see the whole execution and so you can check directly my implementation.
The problem is with custom gradient computation. In fact, here I have to compute the gradient between the input of my layer and its output, in order to backpropagate gradient at the previous layer. 
For now, I found a "hack" solution, in which I computed a gradient with the accepted shape of the previous layer, filled by 0s. 
But surely this is not working, as you can see in loss and accuracy.
My question is: Is implemented a way to compute automatically the gradients? If not, any tip?
Thanks in advance
